pom.xml

```diff
@@ -46,6 +46,7 @@
         <awssdk1.version>1.11.714</awssdk1.version>
         <awssdk1-swf-libs.version>1.11.22</awssdk1-swf-libs.version>
         <awssdk2.version>2.13.58</awssdk2.version>
+        <bouncycastle.version>1.65</bouncycastle.version>
```

Comment:
Camel master has moved on to `1.66` so we should align with that.

Reply:
yes, I agree. I'll change version of bouncycastle.